src/components/molecules/dialog/index.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import Button from '_atoms/button'
+
+import styles from './styles.css'
+
+const Dialog = ({
```

Comment:
the `<Dialog />` component should be a bit more simple, without buttons and texts.